source/reference/method/MongoDBDatabase-withOptions.txt

```diff
@@ -34,6 +34,14 @@ Parameters
        - Type
        - Description
 
+     * - builderEncoder
+       - ``MongoDB\Codec\Encoder``
+       - The encoder used to serialize native PHP types to BSON data in
+         query and aggregation options. If omitted, the ``Database``
+         instance uses the default encoder.
+
+         .. versionadded:: 1.21.1
```

Comment:
This can be 1.21.

[PHPLIB-1646](https://jira.mongodb.org/browse/PHPLIB-1646) only fixed a bug with this method not inheriting the _existing_ Database's option.

If `builderEncoder` is explicitly provided in the `$options` array (which is what this is documenting), it was always passed along to construct the new Database instance.